pkg/services/github.go

```diff
@@ -31,49 +31,51 @@ type GitHubOptions struct {
 }
 
 type GitHubNotification struct {
```

Comment:
If you remove "Status" field it can cause breaking changes for existing customers, why do you need to change it?